bibdata.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- hosts: bibdata
+- hosts: bibdata_staging
```

Comment:
Do we want to rename this playbook to `bibdata_staging.yml`?  I'm not sure if we reached agreement on the naming conventions, but I would be in favor of including the environment name in each filename.